desktop/renderer/index.js

```diff
@@ -3,131 +3,107 @@
  * The main renderer. Holds the global store. When it changes we send it to the main thread which then sends it out to subscribers
  */
 
-import React, {Component} from 'react'
+import React from 'react'
 import ReactDOM from 'react-dom'
-import {Provider} from 'react-redux'
+import {AppContainer} from 'react-hot-loader'
 import configureStore from '../shared/store/configure-store'
-import Nav from '../shared/nav.desktop'
 import injectTapEventPlugin from 'react-tap-event-plugin'
 import ListenLogUi from '../shared/native/listen-log-ui'
-import {reduxDevToolsEnable, devStoreChangingFunctions, setup as setupLocalDebug} from '../shared/local-debug.desktop'
+import {devStoreChangingFunctions} from '../shared/local-debug.desktop'
 import {listenForNotifications} from '../shared/actions/notifications'
+import {bootstrap} from '../shared/actions/config'
+import {updateDebugConfig} from '../shared/actions/dev'
 import hello from '../shared/util/hello'
 
+import Root from './container'
 import {devEditAction} from '../shared/reducers/dev-edit'
 import {setupContextMenu} from '../app/menu-helper'
 
 // For Remote Components
 import electron, {ipcRenderer} from 'electron'
-import RemoteManager from './remote-manager'
 import {ipcLogsRenderer} from '../app/console-helper'
 import loadPerf from '../shared/util/load-perf'
 import merge from 'lodash/merge'
-import {MuiThemeProvider} from 'material-ui/styles'
-import materialTheme from '../shared/styles/material-theme.desktop'
 
-ipcLogsRenderer()
+function setupApp (store) {
```

Comment:
It's easier to read this diff if you ignore whitespace. You can do that by adding `?w=1` to the URL.